### DISCUSSION TITLE
Placeholder label on Polylookup when record hasn't been created yet

### DISCUSSION THREAD

**Discussion by markgomez00** [khoait/DCE.PCF#30] [Ideas] [answered]
Since the behavior within dataverse is that the record needs to be created first before you can actually start relating the N:N records.

Can we have a default placeholder label on the PolyLookup when the record hasn't been created yet, something like "Please create the record first...."

**Comment by khoait** [comment]
Yeah. I'll include this in the next release together with the fix for #29. If output selected items is not enabled, the control should show the message in the create form. Thanks for suggesting!

**Comment by khoait** [comment, marked as answer]
This should be fixed in v1.0.17 in release 1.0.1.1. Please check it out, feel free to open new issues if any.
